projects/lib/src/board/westernboard.h

```diff
@@ -163,17 +187,22 @@ class LIB_EXPORT WesternBoard : public Board
 		QString castlingRightsString(FenNotation notation) const;
 		bool parseCastlingRights(QChar c);
 		CastlingSide castlingSide(const Move& move) const;
-		void setEnpassantSquare(int square);
+		void setEnpassantSquare(int square, int target=0);
 		void setCastlingSquare(Side side,
 				       CastlingSide cside,
 				       int square);
+		/*! Helper for Pawn moves. Returns square offset for the
+		 *  given \a step with orientation \a sign. */
+		inline int getPawnPush( const PawnStep& ps, int sign ) const;
```

Comment:
Our convention doesn't use the "get" prefix unless necessary. I think a better name for this function would be `pawnPushOffset`.